src/components/__tests__/Weather.test.tsx

```diff
@@ -0,0 +1,33 @@
+import axios from 'axios';
+import { fetchWeatherData } from '../../__mocks__/WeatherMocks';
+
+jest.mock('axios');
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+describe('mock api calls', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  test('return forcast Sunny', async () => {
+    mockedAxios.get.mockResolvedValue({
+      data: {
+        result: {
+          weather: {
+            forcast: 'Sunny',
+            max: 28,
+            min: 17,
+            description: 'Clear skys all day with a warm summber breaze ariving in the afternoon',
+          },
+        },
+      },
+    });
+    const mockedData = await fetchWeatherData();
```

Comment:
Should be rendering the Weather element and testing if the data from the Mocked API is being displayed.

At the moment this test is just testing that your mock is returning the values that you mocked at the start of the test